assets/programming_language_extensions.json

```diff
@@ -0,0 +1,2154 @@
+[
+  {
+    "name": "ABAP",
+    "type": "programming",
+    "extensions": [".abap"]
+  },
```

Comment:
Need to check if this is getting compiled into the binary increasing the filesize - if it is maybe we should cut it down to fewer

Reply:
Fixed with next commit, please see @hrishioa